source/templates.tex

```diff
@@ -6978,8 +6977,8 @@
 \pnum
 \begin{note}
 Because the explicit template argument list follows the function
-template name, and because conversion member function templates and
-constructor member function templates are called without using a
+template name, and because
+constructor member function templates\iref{class.ctor} are named without using a
```

Comment:
Instead of "constructor member function templates", I think "constructor templates" would be clearer and easier to understand.

Reply:
Again, pre-existing junk.